### DISCUSSION TITLE
How can I get it to work with localtunnel?

### DISCUSSION THREAD

**Discussion by JohannesPertl** [Waboodoo/HTTP-Shortcuts#445] [Q&A]
I'm using localtunnel to test a simple API. Curl requests work, but in app I always get `511 Network Authentication Required`.

**Comment by Waboodoo** [comment]
I'm not familiar with localtunnel, but it sounds like it requires some kind of authentication or tries to verify the client (in this case curl or the HTTP Shortcuts app) in some way. I couldn't find out anyhing useful with a quick search. Here's an idea though:
Try opening the URL from a browser first, on the same device, and then afterwards try the same URL from the app.

Also, check if the response that you're getting contains any useful information in the body. It might be that localtunnel gives you some details about the error there.
